.github/workflows/release.yaml

```
@@ -24,6 +24,32 @@ jobs:
               }
             })
 
+  build-rom-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 📂 Checkout code
+        uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: 👷 Build prql-compiler
+        run: cargo build --release
```

```suggestion
      - name: 👷 Build prqlc
        run: cargo build --bin prqlc --release
```

Slightly more efficient I think